index.html

```diff
@@ -1489,5 +1489,24 @@ <h3>Further Techniques</h3>
         </p>
       </section>
     </section>
+
+    <section>
+      <h2>Test suite</h2>
+
+      <em>This section is non-normative.</em>
+
+      <p>
+        The W3C Verifiable Claims Working Group has produced a 
+        <a href="https://github.com/w3c/vc-test-suite/">test suite</a> in order 
+        for implementers to validate their conformance with the current specifications. 
+        Conforming implementations are encouraged to pass all tests in the test suite.
```

Comment:
It may be better with this line removed entirely: "Conforming implementations are encouraged to pass all tests in the test suite."

Reply:
I did say that more language was needed...

Reply:
I have removed the entire line until we come up with a better text.